docs/reference/snapshot-restore/apis/delete-snapshot-api.asciidoc

```diff
@@ -0,0 +1,76 @@
+[[delete-snapshot-api]]
+=== Delete snapshot API
+++++
+<titleabbrev>Delete snapshot</titleabbrev>
+++++
+
+Deletes a <<snapshot-restore,snapshot>> of a cluster or specified data streams and
+indices.
+
+////
+[source,console]
+----
+PUT /_snapshot/my_repository
+{
+  "type": "fs",
+  "settings": {
+    "location": "my_backup_location"
+  }
+}
+----
+// TESTSETUP
+////
+
+[source,console]
+-----------------------------------
+DELETE /_snapshot/my_repository/my_snapshot
+-----------------------------------
+// TEST[s/my_snapshot/my_snapshot]
+
+[[delete-snapshot-api-request]]
+==== {api-request-title}
+
+`DELETE /_snapshot/<repository>/<snapshot>`
+
+[[delete-snapshot-api-desc]]
+==== {api-description-title}
+
+Use the delete snapshot API to delete a <<snapshot-restore,snapshot>>, which is a backup taken from a running {es} cluster.
+
+When deleting a snapshot from a repository, {es} deletes all files that are associated with the snapshot and not used by any other snapshots. All files that are shared with at least one other existing snapshot are left intact.
+
+If you attempt to delete a snapshot while it is being created, the snapshot process aborts and all associated will be deleted.
+
+To delete multiple snapshots in a single request, separate the snapshot names with a comma or use a wildcard (`*`).
```

Comment:
Technically, you can do both. `DELETE /_snapshot/my_repository/test1*,*test2*` should be valid.